R/globals.R

```diff
@@ -12,3 +12,6 @@ utils::globalVariables(
     "total_time"
   )
 )
+
+# Setting threshold to debug (temporary and should be removed)
+logger::log_threshold(logger::INFO)
```

Comment:
Is it important in this version?

Reply:
Not really, I'll remove it. I see that there's a different task with this purpuse